src/app/projects/states/dashboard/directives/student-task-list/create-portfolio-task-list-item/create-portfolio-task-list-item.component.scss

```diff
@@ -0,0 +1,7 @@
+status-icon {
+  width: 100%;
+  min-width: 40px;
+  min-height: 40px;
+  font-size: 1.25em;
+  display: inline-block;
+}
```

Comment:
Is there a reason we have this class 3 times? This can be defined once and imported